makefiles/Makefile.docs

```diff
@@ -200,6 +200,7 @@ examples:
 	curl -SfL https://raw.githubusercontent.com/mongodb/mongo-csharp-driver/master/tests/MongoDB.Driver.Examples/TransactionExamplesForDocs/RetryExample3.cs -o ${DRIVERS_PATH}/TransactionsRetryExample3.cs
 	curl -SfL https://raw.githubusercontent.com/mongodb/mongo-csharp-driver/master/tests/MongoDB.Driver.Examples/CausalConsistencyExamples.cs  -o ${DRIVERS_PATH}/CausalConsistencyExamples.cs
 	curl -SfL https://raw.githubusercontent.com/mongodb/mongo-csharp-driver/master/tests/MongoDB.Driver.Examples/TransactionExamplesForDocs/WithTransactionExample1.cs  -o ${DRIVERS_PATH}/withTxnExample1.cs
+	curl -SfL https://raw.githubusercontent.com/mongodb/mongo-csharp-driver/master/tests/MongoDB.Driver.Examples/StableApiExamples.cs  -o ${DRIVERS_PATH}/VersionedApiExamples.cs
```

Comment:
Original filename differs from outputted filename to avoid errors for server docs repo [here](https://github.com/mongodb/docs/blob/f15df6e24dda9e36f7c7029e75921f3b01121b53/source/reference/stable-api.txt#L125). This file also does not appear as an example in the server [docs Makefile](https://github.com/mongodb/docs/blob/f15df6e24dda9e36f7c7029e75921f3b01121b53/Makefile#L113). Going to leave this change as-is for now in hopes that this unblocks the server docs team, but can be changed later on